app/src/main/java/org/mozilla/fenix/search/SearchDialogFragment.kt

```diff
@@ -374,10 +392,12 @@ class SearchDialogFragment : AppCompatDialogFragment(), UserInteractionHandler {
             updateToolbarContentDescription(it.searchEngineSource)
             toolbarView.update(it)
             awesomeBarView.update(it)
+            addVoiceSearchButton(it)
```

Comment:
This reverses the order of the buttons.  From the design QR button should be on the left.